machines/pod_1x1_4X2Y/Makefile.machine.include

```diff
@@ -1,6 +1,8 @@
 BSG_MACHINE_PODS_X									  = 1
 BSG_MACHINE_PODS_Y									  = 1
 
+BSG_MACHINE_XLEN                      = 32
```

Comment:
Set this for all other machines

Reply:
Done, please look at the latest commit, thanks!